### DISCUSSION TITLE
playSMS, Signal, ... Container for Two-Factor Gateway

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#472] [Ideas] [answered]
https://nextcloud-twofactor-gateway.readthedocs.io/en/latest/Admin%20Documentation/

**Comment by szaimen** [comment, marked as answer]
Hello, we will not add this because you should be already able to do this on your own by following this simple documentation.

**Comment by ghost** [comment]
sad🥲, but according to the logic you should be able to set up a Next Cloud with the Next Cloud installation documentation.🤔

**Reply by szaimen** [reply]
I don't ubderstand!

Just follow https://nextcloud-twofactor-gateway.readthedocs.io/en/latest/Admin%20Documentation/ and you are done
